### PR TITLE
[NFC] Update OpenCL installation process

### PR DESCRIPTION
**Pull request by vladimirlaz**
This version supports side-by-side installation of OpenCL RT for
Intel CPU and OpenCL RT for Intel FPGA emulation.

### CODE REVIEW

File: sycl/doc/GetStartedGuide.md [hunk 1]
```diff
@@ -197,6 +193,10 @@ CL_CONFIG_DEVICES = fpga-emu
 2) Create ICD file pointing to the new runtime
 
     ```bash
+    # OpenCL FPGA emulation RT
+    echo  /opt/intel/oclfpgaemu_<fpga_version>/x64/libintelocl_emu.so >
```
Comment (kbobrovs) [inline sycl/doc/GetStartedGuide.md:197]: NIT: this command (and maybe some others) will require root access, could be mentioned somewhere

File: sycl/doc/GetStartedGuide.md [hunk 2]
```diff
@@ -197,6 +193,10 @@ CL_CONFIG_DEVICES = fpga-emu
 2) Create ICD file pointing to the new runtime
```
Comment (vladimirlaz) [inline sycl/doc/GetStartedGuide.md:193]:
```suggestion
2) Create ICD file pointing to the new runtime (requires root access)
```

File: sycl/doc/GetStartedGuide.md [hunk 3]
```diff
@@ -226,7 +236,9 @@ folder:
 5) Configure library paths
```
Comment (vladimirlaz) [inline sycl/doc/GetStartedGuide.md:236]:
```suggestion
5) Configure library paths (requires root access)
```

Comment (vladimirlaz) [inline sycl/doc/GetStartedGuide.md:236]:
```suggestion
5) Configure library paths (requires root access)
```

**Pull request merged**